closure-deps/bin/closuremakedeps.js

```diff
@@ -52,14 +53,15 @@ function parseArgs(args) {
         alias: 'f',
         default: [],
         array: true,
-        description: 'One or more input files to calculate dependencies ' +
-            'for. The namespaces in these files will be combined with ' +
-            'those given with the --root flag to form the set of ' +
-            'namespaces to find dependencies for. If this is a file that ' +
-            'contains repeated calls to goog.addDependency, then the calls ' +
-            'will be used as forward declarations. e.g. if you include ' +
-            'Closure Library\'s deps.js file, then there is no need to ' +
-            'include closure/goog/array/array.js as an input file as well. ' +
+        description:
+            'One or more input files to calculate dependencies for. The ' +
+            'namespaces in these files will be combined with those given ' +
+            'with the --root flag to form the set of namespaces to find ' +
+            'dependencies for. If this is a file that contains repeated ' +
+            'calls to goog.addDependency, then the calls will be used as ' +
+            'forward declarations. e.g. if you include Closure Library\'s ' +
+            'deps.js file, then there is no need to include ' +
```

Comment:
Seems unrelated - revert?

Reply:
Yep, trying to figure out why Travis's clang-format is not cooperating with my local one